.claude/hooks/notify.sh

```
@@ -0,0 +1,103 @@
+# Function to play a sound file with cross-platform support
+play_sound_file() {
+    local sound_file="$1"
+    
+    # Check if file exists
+    if [[ ! -f "$sound_file" ]]; then
+        echo "Warning: Sound file not found: $sound_file" >&2
+        return 1
+    fi
+    
+    # Detect OS and use appropriate command-line audio player
+    local os_type="$(uname -s)"
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)